sashimi-webapp/test/e2e/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "mocha": true
+  },
+  "globals": {
+    "expect": true,
+    "sinon": true
+  },
+  "rules": {
+    "arrow-body-style": "off",
+    "prefer-arrow-callback": "off",
+    "func-names": "off",
+    "no-unused-expressions": "off",
+    "no-console": "off"
+  }
+}
```

Comment:
Yes eslint is complaining that.

Because nightwatch is run using nodejs environment,
1. `console.log` is one way for us to get information from the terminal.
2. the no-unused-expressions is also something needed for the nightwatch + mocha and chai setup. For example:
```
expect.content.to.be.true
```

This, will be viewed by eslint as a unused expression, since it is not assigned back anything and it doesn't look like a function to eslint.